styles/variables.less

```diff
@@ -3,4 +3,3 @@
 
 // App
 // ---------------------------------------
-@agate-app-keepout: 18px;
```

Comment:
This removal may actually cause problems, since some apps import this file and refer to this variable.